src/backend/nodes/copyfuncs.c

```diff
@@ -253,6 +253,8 @@ CopyPlanFields(const Plan *from, Plan *newnode)
 	COPY_BITMAPSET_FIELD(extParam);
 	COPY_BITMAPSET_FIELD(allParam);
 	COPY_NODE_FIELD(flow);
+	COPY_SCALAR_FIELD(locustype);
```

Comment:
Do we also need to implement readfuncs/outfuncs?

Reply:
This is only for PREPARE EXECUTE statement.
Plancache will copy PlannedStmt list if it's not a oneshot plan. 
So, Add files locustype and parallel in CopyPlanFields(). 
Implement Read/Write is ok, but it seems no used for this scenario or others now.